.github/workflows/cifuzz.yml

```
@@ -0,0 +1,35 @@
+name: CIFuzz
+on: [pull_request]
+permissions: {}
+jobs:
+  Fuzzing:
+    runs-on: ubuntu-latest
+    permissions:
+      security-events: write
+    steps:
+    - name: Build Fuzzers
+      id: build
+      uses: google/oss-fuzz/infra/cifuzz/actions/build_fuzzers@master
```

As a security best practice, uses of external GitHub Actions should be pinned to commit hashes.


```suggestion
      uses: google/oss-fuzz/infra/cifuzz/actions/build_fuzzers@431182e97348a158979ab8af46f6a73c374664ba
```